android/app/src/main/res/layout/nav_header.xml

```diff
@@ -5,13 +5,16 @@
     android:gravity="bottom"
     android:orientation="vertical"
     android:padding="4dp"
+    android:background="@color/color_primary"
     android:theme="@style/ThemeOverlay.AppCompat.Dark">
 
     <ImageView
         android:id="@+id/headerDrawer"
         android:layout_width="fill_parent"
         android:layout_height="fill_parent"
         android:contentDescription="@string/header_image"
-        android:src="@drawable/twitter" />
+        android:src="@mipmap/ic_launcher" />
 
 </LinearLayout>
+
```

Comment:
@mananwason sorry for the mistake ,actually i am beginner so i didn't know about that.